src/omv/imlib/jpeg.c

```diff
@@ -1012,8 +1013,12 @@ void JPEG_IRQHandler() {
 }
 
 void jpeg_mdma_irq_handler() {
-    HAL_MDMA_IRQHandler(&JPEG_MDMA_Handle_In);
-    HAL_MDMA_IRQHandler(&JPEG_MDMA_Handle_Out);
+    if (MDMA->GISR0 & (1 << OMV_MDMA_CHANNEL_JPEG_IN)) {
```

Comment:
Doesn't the IRQ handler it's self check the stream ISR? I guess this saves the jump.

Reply:
> I guess this saves the jump.

Exactly.